devicetypes/smartthings/zwave-dual-switch.src/zwave-dual-switch.groovy

```diff
@@ -235,6 +235,6 @@ private crcEncap(physicalgraph.zwave.Command cmd) {
 	zwave.crc16EncapV1.crc16Encap().encapsulate(cmd).format()
 }
 
-def zwaveEvent(physicalgraph.zwave.Command cmd) {
-	log.warn "Unhandled command: ${cmd}"
+def zwaveEvent(physicalgraph.zwave.Command cmd, int endpointId) {
```

Comment:
Would it make more sense to do something like:

```
def zwaveEvent(physicalgraph.zwave. Command cmd, endpoint=null)
```

to match the others? The handling of secure messages just calls `zwaveEvent(cmd)`

Reply:
fixed to unify them